src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -1577,6 +1662,40 @@ class OverlapsClauseSegment(BaseSegment):
     match_grammar = Nothing()
 
 
+@tsql_dialect.segment(replace=True)
+class DescribeStatementSegment(BaseSegment):
+    """A `Describe` statement.
+
+    Not present in T-SQL.
+    """
+
+    type = "describe_statement"
+    match_grammar = Nothing()
+
+
+@tsql_dialect.segment(replace=True)
+class CommentClauseSegment(BaseSegment):
+    """A comment clause.
+
+    Not present in T-SQL.
+    """
+
+    type = "comment_clause"
+    match_grammar = Nothing()
```

Comment:
- Remove reference to this on line 1966 in the CREATE TABLE segment.

- Remove reference to this on line 1293 in the CREATE TABLE segment.

These are the only segments in ANSI where this is referenced so and they are both overwritten in TSQL so if we remove the references listed above then no need to add this stub.

Reply:
@jpy-git removed, thanks!